packages/api/src/routes/medical/schemas/network.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+import { queryMetaSchema } from "../../pagination";
+
+const defaultPageSize = 100;
+export const networkGetSchema = z
+  .object({
+    filter: z.string().optional(),
+  })
+  .and(queryMetaSchema)
+  .transform(data => ({
+    ...data,
+    count: data.count ?? defaultPageSize,
```

Comment:
I don't think the route should define the default page size. Defining both default and maximum are a job of the domain and/or the command, IMO.

Reply:
domain should be just business logic, right? So probably not there because default page size is an impl. detail, so added it to the command. I'm still getting a feel for the responsibilities but this definitely makes sense logically.